src/screens/search.screen.tsx

```diff
@@ -29,7 +29,7 @@ const SearchScreen = ({ navigation }: ScreenProps) => {
               location={name}
               setLocation={
                 place => {
-                  dispatch(setForecast(undefined))
+                  // No need to reset forecast here.
```

Comment:
But if while you wait for the new forecast, will not Main screen display the old forecast?

Reply:
Never mind, saw the git log that explains it.